fbw-a380x/src/wasm/systems/a380_systems/src/hydraulic/mod.rs

```diff
@@ -1639,12 +1642,30 @@ impl A380Hydraulic {
             nose_steering: SteeringActuator::new(
                 context,
                 Angle::new::<degree>(75.),
-                AngularVelocity::new::<radian_per_second>(0.35),
+                AngularVelocity::new::<radian_per_second>(0.2618), // Reference is 15 deg/s
```

Comment:
Why not encode that in the code rather than a comment? (same for the two 4.5 deg/s below)
```suggestion
                AngularVelocity::new::<degree_per_second>(15),
```